spec/catalog_request_spec.rb

```diff
@@ -18,6 +18,8 @@
 
 require 'spec_helper'
 
+require 'vra/http'
```

Comment:
Curious why you had to add this. If you got a LoadError or similar, I wonder if we missed a stub/mock somewhere.